### PR TITLE
Ionesio's codebase cleanup 1

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
Removing duplicated code from `packages/syft/src/syft/core/node/common/node_service/get_all_requests/get_all_requests_service.py`

### CODE REVIEW

File: packages/syft/src/syft/core/node/common/node_service/object_request/object_request_service.py [hunk 1]
```diff
@@ -49,6 +47,8 @@
 from .object_request_messages import CreateRequestResponse
 from .object_request_messages import DeleteRequestMessage
 from .object_request_messages import DeleteRequestResponse
+from .object_request_messages import GetAllRequestsMessage
+from .object_request_messages import GetAllRequestsResponseMessage
 from .object_request_messages import GetBudgetRequestsMessage
```
Comment (rasswanth-s) [inline packages/syft/src/syft/core/node/common/node_service/object_request/object_request_service.py:52]: MAYBE:
in Line 407 keep only a single if
`if not _can_triage_request`
as _requests variable is already assigned to `node.data_requests.all()` in L405

File: packages/syft/src/syft/core/node/domain/client.py [hunk 2]
```diff
@@ -72,7 +72,7 @@ def __init__(self, client: Client) -> None:
     def requests(self) -> List[RequestMessage]:
 
         # relative
-        from ..common.node_service.get_all_requests.get_all_requests_messages import (
+        from ..common.node_service.object_request.object_request_messages import (
             GetAllRequestsMessage,
         )
```
Comment (rasswanth-s) [inline packages/syft/src/syft/core/node/domain/client.py:78]: Q: In L83,84 why do we serialize and de serialize  the same message?
can't we send it directly?
<pre>
<code>
blob = serialize(msg, to_bytes=True)
msg = deserialize(blob, from_bytes=True)
</pre>
</code>

**Review by rasswanth-s — APPROVED** [review]
LGTM

File: packages/syft/src/syft/core/node/common/node_service/object_request/object_request_service.py (hunk 1 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/common/node_service/object_request/object_request_service.py:52]: Solved by  ace647c

File: packages/syft/src/syft/core/node/domain/client.py (hunk 2 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/domain/client.py:78]: Solved by 639ed60

**Comment by IonesioJunior** [comment]
Thank you for the feedback and valuable observations @rasswanth-s !

**Pull request merged**